bundler/spec/commands/newgem_spec.rb

```diff
@@ -1355,6 +1367,10 @@ def create_temporary_dir(dir)
 
     let(:require_relative_path) { "gem" }
 
+    let(:minitest_test_file_path) { "test/test/test_gem.rb" }
+
+    let(:minitest_test_class_name) { "class TestTestGem < Minitest::Test" }
```

Comment:
@deivid-rodriguez @hsbt is this expected testing class name? That's one of the benefits for using test as a suffix, since you can just copy the original class name and append "Test" to it. Prepending doesn't work well using the same logic, mainly when "namespaced" class is used.

Reply:
I see, you were expecting `Test::TestGem`, right? I'm not really sure what people use since I never used the `test_` prefix, but yeah you have a point. Hopefully @hsbt or @unasuke can clarify.

Reply:
@simi  @deivid-rodriguez I'm confusing because gem name contains "test". 🤣 

To clarify, I'm using "foo-bar"  as a gem name for the below examples.

`bundle gem foo-bar --test=minitest` generates class `TestFoo::Bar` in `test/test_foo/bar.rb` (by latest bundler)

This patch generates `TestFooBar` class instead.

What name of class do you expect to generate in "bundle gem" command?

Reply:
I think I would expect `Foo::TestBar` in `test/foo/test_bar.rb`?

Reply:
I commited c946ee6d . @simi @deivid-rodriguez is it expected one?